src/components/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import BaseSvg from './BaseSvg';
+
+const docLinks = [
+  { href: 'https://onchainkit.xyz', title: 'Docs' },
+  { href: 'https://github.com/coinbase/onchainkit', title: 'Github' },
+  { href: 'https://discord.gg/8gW3h6w5', title: 'Discord' },
+  {
+    href: 'https://www.figma.com/community/file/1370194397345450683/onchainkit',
+    title: 'Figma',
+  },
+  { href: 'https://x.com/Onchainkit', title: 'X' },
+];
+
+export default function Footer() {
+  return (
+    <section className="mt-12 mb-6 flex w-full flex-col justify-between gap-2 md:flex-row ">
+      <aside className="flex items-center justify-center md:justify-start">
+        <h3 className="mr-2 text-m">Built with love on</h3>
+        <BaseSvg />
```

Comment:
I think is best we say, `Built with love by the OnchainKit team`. And avoid mentioning Base, as this is a repo done by Coinbase.